src/taipy/gui/gui_actions.py

```diff
@@ -205,20 +205,28 @@ def invoke_long_callback(
     user_status_function_args: t.Union[t.Tuple, t.List] = [],
     period=0,
 ):
-    """Invoke a long running user callback (uses Threads).
+    """Invoke a long running user callback.
+
+    Long-running callbacks are run in a separate thread to not block the application itself.
 
     See the [User Manual section on Long Running Callbacks](../../gui/callbacks/#long-running-callbacks)
     for details on when and how this function can be used.
 
     Arguments:
-        state (State^): The `State^` instance, which is an argument of the callback function.
-        user_function (Callable[[...], None]): The function that will be run independently of taipy gui (cannot use `State^`).
-        user_function_args (Optional[List|Tuple]): The remaining arguments, as a List.
-        user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status function that is invoked at the end (of and possibly during) the runtime of the user_function.<br/>
-            The first parameter of this function **must** be a `State^`. The second parameter of this function **must** be a Union[`bool`, `int`] that will indicate a progression (`int`) or the final status (`bool`)
-        user_status_function_args (Optional[List|Tuple]): The remaining arguments ot the user status function, as a List.
-        period (int): The interval in milli-seconds at which the user_status_function is called (default value is 0).</br>
-            When set to a value < 500, the user_status_function is called only when the user_function is terminated.
+        state (State^): The `State^` instance, as received in any callback.
+        user_function (Callable[[...], None]): The function that will be run independently of Taipy GUI. Note
+          that this function must not use *state*, which is not persisted across threads.
+        user_function_args (Optional[List|Tuple]): The remaining arguments.
+        user_status_function (Optional(Callable[[State^, bool, ...], None])): The optional user-defined status
+            function that is invoked at the end of and possibly during the runtime of *user_function*.<br/>
+            The first parameter of this function **must** be a `State^`. The second parameter of this function
+            **must** be a Union[`bool`, `int`] that will indicate whether a progression (`int`) or the final
+            status (`bool`).
+        user_status_function_args (Optional[List|Tuple]): The remaining arguments of the user status function.
+        period (int): The interval, in milliseconds, at which *user_status_function* is called.<br/>
+            The default value is 0, meaning no call to *user_status_function* will happen.</br>
```

Comment:
0 is the same as < 500: user_status_function will be called only at the end

Reply:
if you don't want calls, set user_status_function to None

Reply:
Right. Let me clarify.